packages/mobile/src/store/offline-downloads/slice.ts

```diff
@@ -124,19 +156,24 @@ const slice = createSlice({
 })
 
 export const {
+  // TODO: don't name these the same thing
```

Comment:
wdym?

Reply:
startDownload and batchStartDownload don't actually use the same states. They should be named differently so they don't sound like the single vs batch of the same action.